configs/katello/nightly.yaml

```diff
@@ -33,3 +33,46 @@
       - centos-sclo-rh-rhel-7
       - centos-7-server-updates
       - centos-7-server
+  - name: katello-nightly-pulp3-rhel7
```

Comment:
Perhaps we should name this `pulpcore` to match other repositories and uses?

Reply:
done